### ISSUE TITLE
Issue with Chapter titles - space being inserted at beginning of every chapter

### ISSUE THREAD

**Issue by jimbo1jimbo** [yermak/AudioBookConverter#483]
When I merge a number of M4A files, AudioBookConverter always inserts a space before the "Chapter Name" displayed on the main screen; e.g, "Chapter 1" on screen becomes " Chapter 1" in the resulting M4B file.

M4A tags populated using MP3Tag. M4B chapter tags scanned using Foobar2000.

**Comment by PepperRobb97** [comment]
Jimbo
I'm unable to use the current version on my pc due to errors with java I believe. However, I can use the older version, 5.6.2 and edit the "properties" file. Doing so allows me to fix this issue. In editing the properties file I was able to not only fix the space issue, but several other smaller things I found others having issues with. Hopefully, this helps you out. I've actually had issues on occasion getting the properties file to populate, but I've attempted to paste a older properties file in it's place and it seems to work....

Extract the 5.6.2 [portable file](https://github.com/yermak/AudioBookConverter/releases/download/version_5.6.2/) anywhere you wish. Place the "properties" file in the following location:

C:\Users\username\AppData\Roaming\AudioBookConverter-5.6.2

Let me know if this helps you in the meantime. Hope this helps you out. 

[AudioBookConverter-5.6.2.zip](https://github.com/yermak/AudioBookConverter/files/12060198/AudioBookConverter-5.6.2.zip)

**Comment by jimbo1jimbo** [comment]
Thanks for coming back to me.

I followed your instructions, but I'm afraid the problem still exists.

I even tried an earlier version (5.5.1) with your (renamed) Properties file
and that didn't work either.

Hopefully you can get it sorted soon

Thanks again
Jim Southon

**Comment by PepperRobb97** [comment]
> Thanks for coming back to me. I followed your instructions, but I'm afraid the problem still exists. I even tried an earlier version (5.5.1) with your (renamed) Properties file and that didn't work either. Hopefully you can get it sorted soon Thanks again Jim Southon
> […](#)

Well I gave it a shot. I'm sorry it didn't work for you. Hopefully Yermak can get things sorted with the new version when there's time (as I'm sure he's busy) and find a solution for everyone's issues. I believe that prior to using my fix, I went back into MP3Tag and setup an action to adjust the title's of every chapter (I utilize actions quite a bit in MP3 tag as a way to  edit metadata and adjust things VERY quickly for things en-mass) Here are 2 examples of removing leading characters in the titles. It's not a fix for THIS software. But maybe it can get you by until it does get fixed? 
[Action - Remove Characters In Title.zip](https://github.com/yermak/AudioBookConverter/files/12062799/Action.-.Remove.Characters.In.Title.zip)

**Comment by jimbo1jimbo** [comment]
Thanks so much for the suggestion - that's really helpful of you.

I've actually been doing precisely that, already.

It fixes the problem, with not a lot of effort, but it would obviously be
better if AudiobookConverter populated the Title tag correctly.

Thanks again
Jim

**Comment by PepperRobb97** [comment]
Absolutely! Great minds and all that. Again, he's obviously got a lot on his plate already. From what I can see, this is something he works on when he can (he has a full time job like the rest of us) and life does get in the way. So unless there are other's out there working on this in collaboration with him, we'll just have to patiently await any fixes. 

I haven't been able to utilize the newer versions as Windows seems to have issues with the program and Java (from looking in the log file) which is why I can't edit the properties file. This is what I'd REALLY like to be able to do. But honestly? Even with the tweaking I've done on my own, the program works great for my needs right now so I'm willing to wait it out (even if I do need to occasionally use MP3Tag to offset some things). I'm just glad that I have Audiobook Converter at all as I can at least utilize the chapter editing. I haven't gotten proficient with working with other languages or getting environments up and running in windows to utilize other tools that they may offer here on GitHub that may potentially do what I'm looking for, so for now? This works. All the rest of us can offer each other is support and advice in the meantime!

**Comment by yermak** [comment]
Same as #633 
Fixed, will be released in next version
